chunky/src/java/se/llbit/chunky/chunk/biome/SimpleQuartBiomeData3d.java

```diff
@@ -0,0 +1,44 @@
+package se.llbit.chunky.chunk.biome;
+
+import java.util.Arrays;
+
+/**
+ * Implementation of a 4*64*4 biome representation
+ */
+public class SimpleQuartBiomeData3d implements BiomeData {
+  public static final int BIOME_DATA3D_HEIGHT = 256 / QUART;
+  public static final int BIOMES_ARRAY_SIZE = QUART * QUART * BIOME_DATA3D_HEIGHT;
+
+  /** internally varies first z -> x -> y */
+  private final int[] biomes = new int[BIOMES_ARRAY_SIZE];
+
+  @Override
+  public int getBiome(int chunkLocalX, int chunkLocalY, int chunkLocalZ) {
+    if(chunkLocalY < 0 || chunkLocalY > 255) {
+      return 0;
+    }
+
+    return biomes[quartIdx(chunkLocalX, chunkLocalY, chunkLocalZ)];
+  }
+
+  @Override
+  public void setBiomeAt(int chunkLocalX, int chunkLocalY, int chunkLocalZ, int biome) {
+    if(chunkLocalY < 0 || chunkLocalY > 255) {
+      return;
+    }
+
+    biomes[quartIdx(chunkLocalX, chunkLocalY, chunkLocalZ)] = biome;
+  }
+
+  private static int quartIdx(int x, int y, int z) {
```

Comment:
```suggestion
  private static int getQuartIndex(int x, int y, int z) {
```

And maybe explain that x,y,z are block positions in the chunk (i.e. 0..15) via javadoc